addons/website/models/ir_http.py

```diff
@@ -133,7 +133,7 @@ def _auth_method_public(cls):
 
     @classmethod
     def _register_website_track(cls, response):
-        if getattr(response, 'status_code', 0) != 200:
+        if getattr(response, 'status_code', 0) != 200 or request.httprequest.headers.get('X-Prefetch'):
```

Comment:
Nitpicking: What do you think about a more generic name for this custom http header? In case we have other flows later not related to prefetching for which we don't want to track?

Reply:
Yes, thanks for the remark, it has the advantage of simplicity, it conveys directly the goal of the header. I have changed X-Prefetch for X-Do-Not-track.

Reply:
I'm a bit confused about that rename...
The Do Not Track header exists (even if now deprecated) with a similar meaning... but a different purpose (avoid tracking from the user point of view, not the server).
Wouldn't it be confusing to use a similar name?

Ref: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/DNT

Reply:
I think that X-Do-Not-Track is ok because it is a custom header (starting with X-) and it is different than DNT.
And in either case, it is the client that informs the server to not track the page (here the server cannot know that the client is prefetching the page).

But indeed, it could be confused with the original meaning of DNT which is an user preference and not added for a technical reason (here prefetch).

So maybe one of the proposition of rdeodoo or maybe: X-Disable-Tracking

Let me know what you prefer. Thanks

Reply:
Anything will do for me :)
I like your proposal tho

Reply:
Like it too :)

Reply:
Thanks, I have then used X-Disable-Tracking.
I have also changed the condition slightly as 
request.httprequest.headers.get('X-Disable-Tracking') == '1'

(before the presence of the header alone was disabling the tracking, for example '0' is True)